src/content/guides/getting-started.mdx

```diff
@@ -24,12 +24,15 @@ contributors:
   - wizardofhogwarts
   - myshov
   - anshumanv
+  - d3lm
 ---
 
 webpack is used to compile JavaScript modules. Once [installed](/guides/installation), you can interact with webpack either from its [CLI](/api/cli) or [API](/api/node). If you're still new to webpack, please read through the [core concepts](/concepts) and [this comparison](/comparison) to learn why you might use it over the other tools that are out in the community.
 
 W> The minimum supported Node.js version to run webpack 5 is 10.13.0 (LTS)
 
+<StackBlitzPreview example="getting-started/basic-setup?terminal=" />
```

Comment:
```suggestion
<StackBlitzPreview example="getting-started?terminal=" />
```
I think this needs an update.

<img width="736" alt="CleanShot 2021-06-10 at 08 02 53@2x" src="https://user-images.githubusercontent.com/1091472/121444527-4990d900-c9c2-11eb-94ec-e54f4db1c260.png">

Reply:
Yep, indeed. Good catch.

Reply:
Yes good catch!

Reply:
Updated.